src/index.ts

```diff
@@ -38,7 +38,9 @@ function yn (value: string | undefined) {
  * Debugging `ts-node`.
  */
 const shouldDebug = yn(process.env.TS_NODE_DEBUG)
-const debug = shouldDebug ? console.log.bind(console, 'ts-node') : () => undefined
+const debug = shouldDebug ?
+  (...args: any) => console.log(`ts-node ${new Date().toISOString()}`, ...args)
```

Comment:
Square brackets need to be re-added.  Otherwise looks good.

Reply:
done :)